orca-web/orca-web.gradle

```
@@ -94,6 +94,16 @@ dependencies {
   testImplementation("io.strikt:strikt-core")
   testImplementation("io.mockk:mockk")
   testImplementation("org.apache.groovy:groovy-json")
+  testImplementation("com.nhaarman:mockito-kotlin:1.6.0")
```

```suggestion
  testImplementation("com.nhaarman:mockito-kotlin")
```
since [kork](https://github.com/spinnaker/kork/blob/6d13c519bb3163fc09d7838df06fead7647fe97b/spinnaker-dependencies/spinnaker-dependencies.gradle#L135) specifies the version.

---

done